services/SecurityService.go

```diff
@@ -0,0 +1,92 @@
+package services
+
+import (
+	"fmt"
+	"github.com/jasongauvin/wikiPattern/models"
+	"golang.org/x/crypto/bcrypt"
+	"time"
+)
+
+type RegisterForm struct {
+	Email    string `form:"userEmail" binding:"required"`
+	Password string `form:"userPassword" binding:"required"`
+}
+
+type LoginForm struct {
+	Email    string `form:"userEmail" binding:"required"`
+	Password string `form:"userPassword" binding:"required"`
+}
+
+// SaveUser creates a user.
+// Return the created object
```

Comment:
We already know the return type thanks to the function signature, redundant comment